openreview/conference/templates/recruit_reviewers_post_process.py

```diff
@@ -54,7 +54,7 @@ def process(client, note, invitation):
 '''.format(REVIEWER_NAME, SHORT_PHRASE)
 
             if (REDUCED_LOAD_INVITATION_NAME):
-                message += 'In case you only declined because you think you cannot handle the maximum load of papers, you can reduce your load slightly. Be aware that this will decrease your overall score for an outstanding reviewer award, since all good reviews will accumulate a positive score. You can request a reduced reviewer load by clicking here: https://openreview.net/invitation?id={}/-/{}&user={}&key={}'.format(CONFERENCE_NAME, REDUCED_LOAD_INVITATION_NAME, user, note.content['key'])
+                message += 'In case you only declined because you think you cannot handle the maximum load of papers, you can reduce your load slightly. Be aware that this will decrease your overall score for an outstanding reviewer award, since all good reviews will accumulate a positive score. You can request a reduced reviewer load by clicking here: https://openreview.net/invitation?id={}/{}/-/{}&user={}&key={}'.format(CONFERENCE_NAME, REVIEWER_NAME, REDUCED_LOAD_INVITATION_NAME, user, note.content['key'])
```

Comment:
Instead of doing a format with three variables, can you rename the `REDUCED_LOAD_INVITATION_NAME` to `REDUCED_LOAD_INVITATION_ID` and use this value only?